### PR TITLE
Minor cleanup

### LINKED ISSUE #3: Repeats the sentence I say instead of the one I said before (German/ de-de)
Well maybe I don't get the point of this skill but mycroft always just repeats the command I give like "Was habe ich gerade gesagt?" (What did I just say?) it answers, "Du hast einfach nur gesagt, was habe ich gerade gesagt" (You just said, what did I just say).

Is that how it's meant to work??

### PR DESCRIPTION
**Pull request by forslund**
- Fix Readme
- Add docstrings
- Fix "what did I say" handling, resolves #3

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -15,41 +15,44 @@
...
 
         def on_speak(message):
             self.last_tts = message.data['utterance']
 
-        if CORE_VERSION_TUPLE >= (18, 2, 1):
-            self.add_event('recognizer_loop:utterance', on_utterance)
-            self.add_event('speak', on_speak)
+        self.add_event('recognizer_loop:utterance', on_utterance)
+        self.add_event('speak', on_speak)
         self.last_stt = self.last_tts = self.translate('nothing')
```
Comment (krisgesling) [inline __init__.py:42]: do we need `self.last_stt` anymore?
Presuming we can remove it and switch the reference of it in `handle_did_you_hear_me` to `self.stt_messages[0]`?

**Review by krisgesling — CHANGES_REQUESTED** [review]
This is great, thanks!

Assuming it's been a while since this was in the Marketplace but if we add some tests it should be good to go.

File: __init__.py (hunk 1 above)
Comment (forslund) [reply __init__.py:42]: You're right I missed this one. Will update shortly

Comment (forslund) [reply __init__.py]: Updated. I also found an issue with the use of the last_stt_time

**Comment by krisgesling** [comment]
Looks great. I am also nicknaming this the Foghorn Leghorn Skill:
> I said, I said, I said, You just said, what time is it

**Review by krisgesling — APPROVED** [review]

**Comment by forslund** [comment]
Lol

**Pull request merged**